src/App.tsx

```diff
@@ -26,70 +27,131 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((good1, good2) => {
```

Comment:
good1 and good2 it's not good naming, you can rename them to firstGood and secondGood
or you can use just default name for sort params, like .sort((a, b) => {